### ISSUE TITLE
bug: broken tags

### ISSUE THREAD

**Issue by carolinewoolard** [orzubalsky/wou#46]
For some reason, the tags for asset mapping don't show up. I keep saving the tags for asset mapping without quotation marks "X" but then I open the practice again and the "x" is back. Why? This same bug applies to ALL of the methods/practices that have tags but they aren't showing up (Daigram Hacking, etc)

![screen shot 2017-09-04 at 10 35 58 am](https://user-images.githubusercontent.com/925548/30030833-f3d4de72-915c-11e7-93db-e80d0f4183da.png)
![screen shot 2017-09-04 at 10 35 49 am](https://user-images.githubusercontent.com/925548/30030835-f3d617d8-915c-11e7-963f-b79000631945.png)
![screen shot 2017-09-04 at 10 35 06 am](https://user-images.githubusercontent.com/925548/30030834-f3d54e98-915c-11e7-94bf-cdab9e0b2f5d.png)

**Comment by orzubalsky** [comment]
This is actually the way it's supposed to work: 

Only tags that have readings associated with them show up, otherwise they would link to a page with no readings.

**Comment by carolinewoolard** [comment]
Confusing! Can all practices / methods have tags? Since we don't show the readings on the front page with the method, the tags seem to be associated with the practice / method. I think it's fine to take people to a page that only shows ALL TAGS, if there's no readings associated with the method.

**Comment by orzubalsky** [comment]
Yes, we can do that but then why have tags? More than half of the tags are not associated with additional content and people wouldn't be able to tell which tags have readings and which ones don't, so readings will be pretty hard to find. 

Maybe if we show all tags everywhere we should make sure to only use a predetermined pool of tags? Like it can only be: 
Forming, Joining, Being In, Listening , Communication, Decision Making, Temperature Checks, Feedback, Facilitation, Group Dynamics, Showing Up, Closing

**Comment by carolinewoolard** [comment]
Yes, the tags should be predetermined! I agree. How about: group functioning, decision-making, role clarification, healing and care, shared leadership, communication, conflict resolution, reflection, analysis of images and systems, and speculative futures? See how I've tried to sort it here: http://studycollaboration.com/

**Comment by carolinewoolard** [comment]
Or we can go with this Forming, Joining, Being In, Listening , Communication, Decision Making, Temperature Checks, Feedback, Facilitation, Group Dynamics, Showing Up, Closing

**Comment by orzubalsky** [comment]
The list in the intro text is great! So all of the tags for each practice and reading would have to be edited/updated...

**Comment by carolinewoolard** [comment]
I know :/ BLAH. But I am willing to do it.

**Comment by carolinewoolard** [comment]
Can you make the tags a drop down menu, so Sam also knows to use them?

**Comment by orzubalsky** [comment]
Yeah, good idea!
